src/js/actions/GuideActions.js

```diff
@@ -1,24 +1,29 @@
-import AppDispatcher from "../dispatcher/AppDispatcher";
-import GuideConstants from "../constants/GuideConstants";
-
-const GuideActions = {
-  /**
-   * @param {String} id to ignore
-   */
-  ignore: function (id) {
-    AppDispatcher.dispatch({
-      actionType: GuideConstants.ORG_IGNORE, id
-    });
-  },
-
-  /**
-   * @param {String} id to follow
-   */
-  follow: function (id) {
-    AppDispatcher.dispatch({
-      actionType: GuideConstants.ORG_FOLLOW, id
-    });
-  }
-};
-
-export default GuideActions;
+var Dispatcher = require("../dispatcher/Dispatcher");
+
+var GuideActions = (function (_Dispatcher) {
+  function _GuideActions () { }
+
+  _GuideActions.prototype.ignore = function ignore (we_vote_id) {
```

Comment:
could you not have the `_GuideActions` object that you can then use `Object.assign` to add to the prototype?